use_case_examples/cifar_brevitas_training/trainer.py

```diff
@@ -90,6 +91,61 @@ def accuracy(
         return list_topk_accs
 
 
+def get_train_set(dataset: str, datadir: Union[str, Path]) -> Dataset:
```

Comment:
It doesn't concern this PR, but we have a lot of redundancy between cifar_training and cifar_fietuning. Such as `get_train/test_set` in equivalent to `get_torchvision_dataset` in cifar_finetuning.

We should discuss the refactoring in the next meeting.